Makefile.rule

```diff
@@ -157,7 +165,8 @@ NO_AFFINITY = 1
 
 # If any gemm arguement m, n or k is less or equal this threshold, gemm will be execute
 # with single thread. You can use this flag to avoid the overhead of multi-threading
```

Comment:
This documentation is not quite correct, since this threshold is now used a multiplier to a per function constant and has nothing to do directly with particular matrix dimensions.

Reply:
That's right, though as far as I tried to understand the transition the idea was that it was closer to the originally intended effect. Not sure how to phrase this, unless we want to call it a controversial
magic number with dubious real-world equivalent :grinning: